design/cli-install-changes.md

```diff
@@ -0,0 +1,426 @@
+        --label-columns stringArray                       a comma-separated list of labels to be displayed as columns
+        --labels mapStringString                          labels to apply to the volume snapshot location
+        --provider string                                 name of the volume snapshot provider (e.g. aws, azure, gcp)
+        --show-labels                                     show labels in the last column
+        --credentials mapStringString                     sets the list of name of the corresponding credentials secret for providers. Format is (provider1:credentials-secret-name1,provider2:credentials-secret-name2,...) (NEW)
+
+      get                                                 Display snapshot locations
+        --default                                         list of unique volume providers and default volume snapshot location (provider1:location-01,provider2:location-02,...) (NEW -- was `server --default-volume-snapshot-locations`))
+```
+
+4) `velero plugin`
+Configuration for plugins.
+
+```
+      add stringArray                           IMAGES [flags] - add plugin container images to install into the Velero Deployment
+
+      get                                       get information for all plugins on the velero server (was `get`)
+        --timeout duration                      maximum time to wait for plugin information to be reported (default 5s)
+
+      remove                                    Remove a plugin [NAME | IMAGE] 
+
+      set
+        --credentials-file mapStringString      configuration to use for creating a secret containing the AIM credentials for a plugin provider. Format is provider:path-to-file. (was `secret-file`)
+        --no-secret                             flag indicating if a secret should be created. Must be used as confirmation if create --secret-file is not provided. Optional. (MOVED FROM install -- not sure we need it?)
+        --sa-annotations mapStringString        annotations to add to the Velero ServiceAccount for GKE. Add iam.gke.io/gcp-service-account=[GSA_NAME]@[PROJECT_NAME].iam.gserviceaccount.com for workload identity. Optional. Format is key1=value1,key2=value2
+        --cacert-file mapStringString           configuration to use for creating a secret containing a custom certificate for an S3 location of a plugin provider. Format is provider:path-to-file. (NEW)
+```
+
+#### Example
+
+Considering this proposal, let's consider what a high-level documentation for getting Velero ready to do backups could look like for Velero users:
+
+After installing the Velero CLI:
+```
+velero config server [flags] (required)
+velero plugin add IMAGES [flags] (add/config provider plugins)
+velero backup-location/snapshot-location create NAME [flags] (run `velero plugin --get` to see what kind of plugins are available; create locations)
+velero backup/restore/schedule create/get/delete NAME [flags]
+velero config restic [flags]
```

Comment:
`velero config restic` probably comes earlier in the workflow, either right after `velero config server`, or just before creating any backups/etc.